src/redux-store/reducers/user-reducer.js

```diff
@@ -0,0 +1,35 @@
+import * as types from "../actions/actionTypes";
+
+const initialLoginState = {
+  username: "",
+  password: ""
+};
+
+export const loginReducer = (state = initialLoginState, action) => {
+  switch (action.type) {
+    case types.LOGIN_START:
+      return {};
+    default:
```

Comment:
Also, set it up so that the user object is set to state after logging in i.e you take whatever details you get about the user from the backend and set it to state so we can render things like the user's name on the profile page.